src/app/components/Drawer.jsx

```diff
@@ -48,13 +48,21 @@ const Select = styled.select`
   color: #7364f0;
   font-size: 16px;
   font-weight: 600;
+
+  :hover{
```

Comment:
need to be `cursor: pointer` directly without `hover`

Reply:
Yes, boss!